pdns/slavecommunicator.cc

```diff
@@ -756,6 +773,7 @@ void CommunicatorClass::slaveRefresh(PacketHandler *P)
   }
   if(rdomains.empty()) { // if we have priority domains, check them first
     B->getUnfreshSlaveInfos(&rdomains);
+    L<<Logger::Info<<"No domains explicitely queued for freshness checks. Checking "<<rdomains.size()<<" unfresh slave zones reported by backend"<<endl;
```

Comment:
https://github.com/jsoref/pdns/commit/2b023c9666b3298dfc6caaa7503c63fb45218a4c#commitcomment-39230791
```suggestion
    L<<Logger::Info<<"No domains explicitly queued for freshness checks. Checking "<<rdomains.size()<<" unfresh slave zones reported by backend"<<endl;
```